without-tax-task.js

```diff
@@ -0,0 +1,10 @@
+/*Napisz funkcję withoutTax(amount), która zwróci kwotę wynagrodzenia po odjęciu
+podatku dochodowego w wysokości 18 % .Wynik zaokrąglij do pełnych złotych.*/
+
+
+function withoutTax(amount) {
```

Comment:
It would be good to have some constant like TAX_PERCENT = 18.

Then it will be very easy to change this value. it's a good practice.